projects/plugins/jetpack-boost/autoload-lib.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Custom autoloader for Jetpack Boost.
+ *
+ * @package automattic/jetpack-boost
+ */
+
+/**
+ * This will make PHP look for \Automattic\Jetpack_Boost in the `jetpack-boost/app` directory.
```

Comment:
It's unfortunate WordPress doesn't follow PSR4. But Jetpack and other plugins seem to manage to get by with Composer's autoloader configured like
```
"autoload": {
    "classmap": [
        "app/"
    ]
}
```

Reply:
Removed the file and went with ^ in 75ce4ed04